src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -401,6 +401,10 @@
       <_EmccLinkStepArgs Include="-o &quot;$(_WasmIntermediateOutputPath)dotnet.js&quot;" />
       <_WasmLinkDependencies Include="$(_EmccLinkRsp)" />
 
+      <!-- sync EXPORTED_RUNTIME_METHODS with src\mono\wasm\wasm.proj -->
+      <_EmccLinkStepArgs Condition="'$(EmccExtraExportedRuntimeMethods)' != ''" 
```

Comment:
Why the condition?

Reply:
This would be better as an itemgroup, since it's a list of names.

Reply:
Ah yeah, better than a property and then duplicates can be removed easily.